README.md

```diff
@@ -26,7 +26,41 @@ To run the full unit test suite, run:
 This will run the same tests that we run in the Travis CI build.
 
 ### Deploying
-To deploy a personal stack:
+First, take a look at what Blox will put in your personal stack by running the
+`showStackConfig` task:
+
+```
+$ ./gradlew showStackConfig
+
+> Task :showStackConfig
+Blox deployment stack configuration:
+
+  Default resource name         (blox.name): blox-buys-alpha-us-west-2 (default)
```

Comment:
No, the defaults use `$USER`. I had to use something in the output, and I just copy/pasted these.